src/index.ts

```diff
@@ -149,23 +173,30 @@ export class Client {
     });
 
     if (resp.status !== 200) {
+      let code;
       let message = resp.statusText;
       try {
-        let response = await resp.json();
-        message = response.message;
+        ({ code, message } = (await resp.json()) as APIError);
       } catch (_) {}
 
+      console.error(
```

Comment:
I assume we'll want to safely log. Doesn't seem like we do this in the library already though. I did notice a `console.debug` statement unchecked.

Reply:
I recently told Jason to add a console log but went out of my way to mention that we should check that it exists first. However, upon seeing this, I went to check that console was actually a thing which one should consider unreliable.

https://developer.mozilla.org/en-US/docs/Web/API/console
According to what I'm seeing here, console is shockingly well supported. The only properties which have poor support are very new things like console.table.